google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java

```diff
@@ -79,9 +81,9 @@ public static void main(String[] args) {
       // into the code here. See the parameter class definition above for descriptions.
       params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
       params.adGroupId = Long.parseLong("INSERT_AD_GROUP_ID_HERE");
-
-      // Optional: Specify a customizerAttributeName. The default value in this example is "Price".
-      params.customizerAttributeName = "Price";
```

Comment:
this line (setting the default value) is never executed when running the example by providing only **customer_id** and **ad_group_id** parameters.

Reply:
In the example you linked to, the line corresponding to line 84 is commented out. I think we should have the same behavior here to make it more obvious to users who are entering the params manually in the class file rather than using the command line.

Reply:
Sorry, that was hidden in GitHub, so I thought you'd only deleted the lines.